docs/coding.rst

```diff
@@ -689,14 +691,39 @@ Because Split styles are not global, we still have to import the page-specific P
     $brand-theme: 'firefox';
 
     @import '~@mozilla-protocol/core/protocol/css/includes/lib';
-    @import '~@mozilla-protocol/core/protocol/css/components/split';
-    @import '~@mozilla-protocol/core/protocol/css/components/picto';
+    @import '~@mozilla-protocol/core/protocol/css/components/billboard';
+
+
+Import CSS Bundles
+~~~~~~~~~~~~~~~~~~
+We created pre-built CSS bundles to be used for said components due to their frequency of use. This method only requires an import into the HTML page. Since it’s already a CSS bundle, we don’t need to import it in the respective CSS page.
+The CSS bundle import only works for the following components:
+
+* Split
+* Card
+* Picto
+* Callout
+* Article
+* Newsletter form
+* Emphasis box
+
+To import a respective bundle, all that is required to place the CSS bundle into the ``page_css`` block like the following:
+
+.. code-block:: html
+
+    {% block page_css %}
+        {{ css_bundle('protocol-split') }}
+        {{ css_bundle('protocol-card') }}
+    {% endblock %}
```

Comment:
Should we add an example of a page bundle here as well, since most pages will have one? Like `{{ css_bundle('my-example-page) }}`

Though if we add that we should also call it out above, something like "Include a CSS bundle in the template's ``page_css`` block along with any other page-specific bundles:"

Reply:
Good idea!